src/Stratis.FederatedPeg.Features.FederationGateway/Models/BlockTipModel.cs

```diff
@@ -10,14 +12,21 @@ namespace Stratis.FederatedPeg.Features.FederationGateway.Models
     /// </summary>
     public class BlockTipModel : RequestModel, IBlockTip
     {
-        public BlockTipModel(uint256 hash, int height)
+        public BlockTipModel(uint256 hash, int height, int matureConfirmations)
         {
             this.Hash = hash;
             this.Height = height;
+            this.MatureConfirmations = matureConfirmations;
         }
 
-        public uint256 Hash { get; }
+        [Required(ErrorMessage = "Block Hash is required")]
+        [JsonConverter(typeof(UInt256JsonConverter))]
+        public uint256 Hash { get; set; }
 
-        public int Height { get; }
+        [Required(ErrorMessage = "Block Height is required")]
+        public int Height { get; set; }
+
+        [Required(ErrorMessage = "Mature Confirmations is required")]
+        public int MatureConfirmations { get; set; }
```

Comment:
Summary not in sync with properties used in this class.

Reply:
Done.